src/modules/dashboard/lists/BorrowAssetsList/BorrowAssetsList.tsx

```diff
@@ -87,27 +91,96 @@ export const BorrowAssetsList = () => {
             availableBorrowsInUSD !== '0.00' && totalLiquidityUSD !== '0'
         );
 
+  if (sortDesc) {
+    if (sortName === 'symbol') {
+      borrowReserves.sort((a, b) => (a.symbol.toUpperCase() < b.symbol.toUpperCase() ? -1 : 1));
+    } else {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      borrowReserves.sort((a, b) => a[sortName] - b[sortName]);
+    }
+  } else {
+    if (sortName === 'symbol') {
+      borrowReserves.sort((a, b) => (b.symbol.toUpperCase() < a.symbol.toUpperCase() ? -1 : 1));
+    } else {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      borrowReserves.sort((a, b) => b[sortName] - a[sortName]);
+    }
+  }
+
   const head = [
-    <AvailableTooltip
-      capType={CapType.borrowCap}
-      text={<Trans>Available</Trans>}
-      key="Available"
-      variant="subheader2"
-    />,
-    <VariableAPYTooltip
-      text={<Trans>APY, variable</Trans>}
-      key="APY_dash_variable_ type"
-      variant="subheader2"
-    />,
-    <StableAPYTooltip
-      text={<Trans>APY, stable</Trans>}
-      key="APY_dash_stable_ type"
-      variant="subheader2"
-    />,
+    {
+      title: <Trans>Asset</Trans>,
+      sortKey: 'symbol',
+    },
+    {
+      title: (
+        <AvailableTooltip
+          capType={CapType.borrowCap}
+          text={<Trans>Available</Trans>}
+          key="availableBorrows"
+          variant="subheader2"
+        />
+      ),
+      sortKey: 'availableBorrows',
+    },
+
+    {
+      title: (
+        <VariableAPYTooltip
+          text={<Trans>APY, variable</Trans>}
+          key="variableBorrowAPY"
+          variant="subheader2"
+        />
+      ),
+      sortKey: 'variableBorrowAPY',
+    },
+    {
+      title: (
+        <StableAPYTooltip
+          text={<Trans>APY, stable</Trans>}
+          key="stableBorrowAPY"
+          variant="subheader2"
+        />
+      ),
+      sortKey: 'stableBorrowAPY',
+    },
   ];
 
+  const renderHeader = () => {
```

Comment:
Lately we've switched approached to how we write these dynamic JSX elements to be rendered. Instead of doing this as a function, we've converted these into `JSX.Element`s and render accordingly.

Roughly it'd look like this:

```js
const ListHeaderComponent: React.FC = () => {
 return (
   ...
 )
}
```

A more complex example can be seen [here](https://github.com/aave/interface-gho/blob/main/src/modules/reserve-overview/Gho/GhoDiscountCalculator.tsx#L220).  This was previously written as a function called `renderHelperText()`.  The reason the function approach isn't best and a JSX component is better is that it won't cause a new invocation on each rerender.  If it's a JSX component, it'll just defer to react to handle any changes in the shadow dom and rerender if so, rather than invoking regardless.

Reply:
updated in 3b6ecbb6ead3f77234fd8f05c8749bbf5da69a4d